content/SCALE/Apps/Collabora.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Collabora App"
+weight: 35
+---
+
+{{< toc >}}
+
+The SCALE **Apps** catalogue now includes [Collabora](https://nextcloud.com/collaboraonline/) from the developers of [Nextcloud](https://nextcloud.com/).
+
+With Collabora, you can host your online Office Suite at home.
```

Comment:
Why Office Suite instead of office suite? I cannot find a reference to this in the Chia documentation so I don't know why this is init-capped.